SPECS/mariner-rpm-macros/macros.go-rpm

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2015-2018 Jan Chaloupka <jchaloup@redhat.com>,
```

Comment:
There are a bunch of files in the `go-rpm-macros` package.
Why are we picking only this one specifically.